saleor/core/tracing.py

```diff
@@ -14,3 +17,12 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+@contextmanager
+def traced_atomic_transaction():
+    with transaction.atomic():
+        with opentracing.global_tracer().start_active_span("transaction") as scope:
+            span = scope.span
+            span.set_tag(opentracing.tags.COMPONENT, "db")
```

Comment:
I'm not sure that's the correct component.

Reply:
Me neither. It will affect how much time is calculated to be spent "in the database."

Reply:
Maybe a span tagged as component `orm`